src/main/java/com/microsoft/aad/adal4j/AdalAuthorizatonGrant.java

```diff
@@ -62,6 +62,12 @@
         if (this.params != null) {
             outParams.putAll(this.params);
         }
+        
+        if(!outParams.containsKey("scope"))
+        {
+        	outParams.put("scope", "openid");

```

Comment:
there are no extra QP supported on token endpoint. remove the if contains check is the way to go.